frontend/src/components/reviews-replies/ReviewOptions.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+<div>
+    <h4 class="options-header">
+        <slot></slot>
+    </h4>
+    <div class="options">
```

Comment:
How hard is it to make the buttons look good with different numbers of options? I noticed that adding a 4th one went over to the next line and still took up a third of the page

Reply:
That's because of the template 1fr 1fr 1fr